transport/transport-runtime/src/main/java/com/google/android/datatransport/runtime/scheduling/jobscheduling/SchedulerConfig.java

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.android.datatransport.runtime.scheduling.jobscheduling;
+
+import javax.inject.Inject;
+
+public class SchedulerConfig {
+
+  private long delta;
+  private long maxAllowedTime;
+  private long maximumDelay;
+
+  @Inject
+  public SchedulerConfig(long delta, long maxAllowedTime, long maximumDelay) {
```

Comment:
An `@Autovalue.Builder` would be nice as well, because `new SchedulerConfig(1,2,3)` is not very readable at call sites.